charts/opa/values.yaml

```diff
@@ -139,7 +139,7 @@ mgmt:
 # NOTE IF you use these, remember to update the RBAC rules below to allow
 #      permissions to get, list, watch, patch and update configmaps
     enabled: false
-    namespaces: [opa, kube-federation-scheduling-policy]
```

Comment:
Just for clarification there is no default namespaces to load policies from. How does this work with what's happening in the code [here](https://github.com/open-policy-agent/kube-mgmt/blob/master/cmd/kube-mgmt/main.go#L84) ?

Reply:
Chekc deployment template, if Helm values are empty / unset,  it sets `--policies` to the namespace where release was installed.

Reply:
Ok. Thanks for the clarification.